### PR TITLE
SunglassesKhronos

### PR DESCRIPTION
**Pull request by echadwick-artist**
This is the first of four contracted "Hero Assets" for Khronos Group's 3D Commerce, a pair of sunglasses with an iridescent coating and featuring:

* Efficient 3D modeling.
* Face-weighted vertex normals for clean shading and optimal compression.
* Optimal texture packing.
* glTF extensions iridescence, transmission, and volume.
* No ambient occlusion applicable since it is mostly metal and glass.
* Extensive tutorial for asset creation in the readme.

screenshot_Large

### CODE REVIEW

File: Models/SunglassesKhronos/README.body.md [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+
+Copyright metadata was added using [Microsoft Visual Studio Code](https://code.visualstudio.com) with the [Cesium glTF Tools Extension](https://marketplace.visualstudio.com/items?itemName=cesium.gltf-vscode).
+
+After conversion, the resulting glTF was then optimized with [RapidPipeline 3D Processor](https://docs.rapidpipeline.com/docs/componentDocs/3dProcessor/3d-processor-overview) to compact the data by removing unneeded information. In this case no decimation was required since the model uses only 13,396 triangles. 
+
+A variant with lossy compression was created with another run of RapidPipeline 3D Processor, applying Draco geometry compression and WEBP texture compression. This compressed the glTF from 371,212 bytes down to 63,146 bytes, a 83% reduction in size.
```
Comment (lexaknyazev) [inline Models/SunglassesKhronos/README.body.md:161]: nit: spell WEBP as WebP.

File: Models/SunglassesKhronos/metadata.json [hunk 2]
```diff
@@ -0,0 +1,47 @@
+{
+    "version": 2,
+    "legal": [
```
Comment (lexaknyazev) [inline Models/SunglassesKhronos/metadata.json:3]: Should there be explicit copyright info for all the images (especially photos) used in the readme?

Comment (echadwick-artist) [reply Models/SunglassesKhronos/metadata.json:3]: I'm not sure. We have photos in other readmes, without explicit licensing. For example https://github.com/KhronosGroup/glTF-Sample-Assets/tree/main/Models/AnisotropyBarnLamp

Comment (echadwick-artist) [reply Models/SunglassesKhronos/metadata.json:3]: Also https://github.com/KhronosGroup/glTF-Sample-Assets/tree/main/Models/GlamVelvetSofa#reference-photos
and https://github.com/KhronosGroup/glTF-Sample-Assets/tree/main/Models/SheenChair#khr_materials_sheen
and https://github.com/KhronosGroup/glTF-Sample-Assets/tree/main/Models/StainedGlassLamp#reference-photos

**Pull request merged**